spring-gateway-ocd/README.md

```diff
@@ -0,0 +1,12 @@
+Running Spring Gateway OCR
```

Comment:
OCD states for one click deployment. Please use just spring-gateway instead spring-gateway-ocd